docs/src/tut/rsvp/index.md

```diff
@@ -790,10 +790,13 @@ This is representative of a user interface that shows the Host each registration
 
 ```
 load: /examples/rsvp-6-vevt/index.mjs
-md5: 6c9ca7f7e8a9cb738005b61cabc1e8b0
-range: 57-67
+md5: 65e69ecdbcea0eab0d04838833ddea55
+range: 59-70
 ```
 
+We've added a gas limit using `{!js} stdlib.setGasLimit` for testing on Ethereum.
+This prevents `{!rsh} API`s from expending all of their tokens on gas fees.
```

Comment:
This is not accurate. There is always a gas limit, even if we don't specify anything.

Normally the gas limit is computed by looking at the contract code, but it is not always accurate. What we're doing here is setting the "limit" to the largest possible amount of gas that could ever be spent. Thus, we're really _removing_ the limit by setting it to be "infinity".

Reply:
Got it!